CMakeLists.txt

```diff
@@ -107,7 +107,7 @@ set(required_deps Cdio
                   FFMPEG
                   Fmt
                   FreeType
-                  Fribidi
+                  FriBidi
```

Comment:
I think you now need to rename

https://github.com/xbmc/xbmc/blob/master/cmake/modules/FindFribidi.cmake

to `cmake/modules/FindFriBidi.cmake`

on Linux.